pkg/diff/test.sql

```diff
@@ -0,0 +1,10001 @@
+INSERT INTO `test`.`testa` VALUES
```

Comment:
how this file come,  why not generate the test data in your test

Reply:
done, use importer to generate data